### PR TITLE
Remove insider program requirement

### PR DESCRIPTION
**Pull request by nishitha-burman**
Remove insider program requirement from WebView2 on HoloLens 2 docs.

AB#45125531

### CODE REVIEW

File: microsoft-edge/webview2/get-started/hololens2.md
Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/hololens2.md:1]: @nishitha-burman -- webview2/roadmap.md says "The WebView2 Preview is available in the Insider Preview for Microsoft HoloLens.", remove

Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/hololens2.md:1]: @nishitha-burman -- webview2/get-started/winui2.md - "Get started with WebView2 in WinUI 2 (UWP) apps" - scrub:

https://learn.microsoft.com/en-us/microsoft-edge/webview2/get-started/winui2#step-5---install-the-winui-2-sdk-microsoftuixaml -- 
""The WebView2 Preview is available in the Insider Preview for Microsoft HoloLens. To access this preview, you must be enrolled in the Windows Insider Program; see [Start receiving Insider builds](https://learn.microsoft.com/en-us/hololens/hololens-insider#start-receiving-insider-builds) in Insider preview for Microsoft HoloLens. See also [Update HoloLens 2](https://learn.microsoft.com/en-us/hololens/hololens-update-hololens)."

Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/hololens2.md:1]: todo: remove the last two sentences about insider preview

Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/hololens2.md:1]: this comment can be Resolved

Comment (mikehoffms) [reply microsoft-edge/webview2/get-started/hololens2.md:1]: this comment can be Resolved

**Review by mikehoffms — APPROVED** [review]

File: microsoft-edge/webview2/get-started/winui2.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ Next, you install the **Microsoft.UI.Xaml** package for this project.  Microsoft
 
    For version 2.8.0 or later, in the **Dependencies** section at the bottom, **Microsoft.Web.WebView2** is listed.
 
-   For HoloLens 2 development, the **Microsoft.Web.WebView2** package must be version 1.0.1722.45 or higher, which may be higher than the default.  The WebView2 Preview is available in the Insider Preview for Microsoft HoloLens.  To access this preview, you must be enrolled in the Windows Insider Program; see [Start receiving Insider builds](/hololens/hololens-insider#start-receiving-insider-builds) in _Insider preview for Microsoft HoloLens_.  See also [Update HoloLens 2](/hololens/hololens-update-hololens).
+   For HoloLens 2 development, the **Microsoft.Web.WebView2** package must be version 1.0.1722.45 or higher, which may be higher than the default.  WebView2 on HoloLens 2 is in Preview and subject to change before general availability. WebView2 is only supported on HoloLens 2 devices running the Windows 11 update. For more information, see [Update HoloLens 2](/hololens/hololens-update-hololens).
```
Comment (mikehoffms) [inline microsoft-edge/webview2/get-started/winui2.md:199]:
```suggestion
   For HoloLens 2 development, the **Microsoft.Web.WebView2** package must be version 1.0.1722.45 or higher, which may be higher than the default.  WebView2 on HoloLens 2 is in Preview and is subject to change before general availability. WebView2 is only supported on HoloLens 2 devices running the Windows 11 update. For more information, see [Update HoloLens 2](/hololens/hololens-update-hololens).
```

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**